From c4f60a77cdba6b2eba19ceae62bbdcca407fe418 Mon Sep 17 00:00:00 2001
From: Sergio Pena <sergio.pena@cloudera.com>
Date: Wed, 19 Oct 2016 15:49:28 -0500
Subject: [PATCH 0788/1164] CDH-46163 HIVE-15022: Missing
 hs2-connection-timed-out in BeeLine.properties
 (Peter Vary, reviewed by Sergio Pena)

(cherry picked from commit 6d2eb1c4126b0a0f02a2b1788448c5ed70d80373)

Change-Id: I3073c765568d9d51e17b754fa9bbbdc153c2e67e
---
 beeline/src/main/resources/BeeLine.properties |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/beeline/src/main/resources/BeeLine.properties b/beeline/src/main/resources/BeeLine.properties
index 52f080a..421bcf7 100644
--- a/beeline/src/main/resources/BeeLine.properties
+++ b/beeline/src/main/resources/BeeLine.properties
@@ -147,7 +147,7 @@ hs2-unexpected-end-of-file: Unexpected end of file when reading from HS2 server.
 cause might be too many concurrent connections. Please ask the administrator to check the number \
 of active connections, and adjust hive.server2.thrift.max.worker.threads if applicable.
 hs2-could-not-open-connection: Could not open connection to the HS2 server. Please check the \
-server URI and if the URI is correct, then ask the administrator to check the server status.\
+server URI and if the URI is correct, then ask the administrator to check the server status.
 hs2-connection-timed-out: Connection timeout when communicating with HS2 server.
 hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thrift server.
 hs2-unexpected-error: Unexpected HS2 error when communicating with the Thrift server.
-- 
1.7.9.5

